src/app/src/components/DownloadFacilitiesButton.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react';
+import { arrayOf, func, string } from 'prop-types';
+import { connect } from 'react-redux';
+import Button from '@material-ui/core/Button';
+
+import { toast } from 'react-toastify';
+
+import { logDownload } from '../actions/logDownload';
+
+const downloadFacilitiesStyles = Object.freeze({
+    listHeaderButtonStyles: Object.freeze({
+        height: '45px',
+        margin: '5px 0',
+    }),
+});
+
+function NonVectorTileFilterSidebarFacilitiesTab({
```

Comment:
For consistency we should rename this to `DownloadFacilitiesButton` to match this to the name of the file.